atomic_reactor/util.py

```diff
@@ -1454,30 +1454,21 @@ def get_log_files(self, osbs, build_id):
 
         # Collect logs from server
         try:
-            logs = osbs.get_build_logs(build_id)
+            logs = osbs.get_build_logs(pipeline_run_name)
```

Comment:
So the pipeline is getting its own logs while it's still running? Is it expected that the logs will only contain messages written up to the point when this executes?

I guess that's probably the original behavior, but god that's weird.

Reply:
Now that you mentioned it, that does seem weird. The code currently and before fetched the logs up to the current point of execution. It does not even seem to be the last plugin to run, which could mean that some information could be lost.

Reply:
Yes, this is how things are working. You cannot upload logs to koji build after, so it will just contain truncated logs